src/Updater/PuliStrategy.php

```diff
@@ -138,4 +139,29 @@ public function getStability()
     {
         return $this->stability;
     }
+
+    /**
+     * Get a remote file content securely using composer/ca-bundle to find the local ca-file.
+     *
+     * @param string $remoteUrl
+     * @return string
+     */
+    private function secureFileGetContents($remoteUrl)
+    {
+        $options = array(
+            'http' => array(
+                'method' => 'GET'
+            )
+        );
+
+        $caPath = CaBundle::getSystemCaRootBundlePath();
+
+        if (is_dir($caPath)) {
+            $options['ssl']['capath'] = $caPath;
+        } else {
+            $options['ssl']['cafile'] = $caPath;
+        }
+
+        return file_get_contents($remoteUrl, false, stream_context_create($options));
```

Comment:
Actually I think this is OK. humbug_get_contents does the same but instead of using CaBundle, it uses its own logic. The result is the same, we simply are using a more updated CaBundle.